types/hellosign-embedded/index.d.ts

```diff
@@ -1,272 +1,213 @@
-// Type definitions for hellosign-embedded v1.2.0
-// Project: https://github.com/HelloFax/hellosign-embedded
+// Type definitions for hellosign-embedded v2.0.0
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `Type definitions for hellosign-embedded 2.0`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!